test/junit/scala/tools/nsc/interpreter/ScriptedTest.scala

```diff
@@ -65,7 +65,6 @@ class ScriptedTest {
   }
   @Test def `t8422 captured multi i/o`() = {
     import java.io.{ StringWriter, StringReader }
-    import scala.compat.Platform.EOL
```

Comment:
FWIW, I see a common idiom in the code base: `import java.lang.System.{lineSeparator => EOL}`.

Reply:
thanks